website/cue/reference/configuration.cue

```diff
@@ -711,7 +711,7 @@ configuration: {
 			title: "Location"
 			body: """
 				The location of your Vector configuration file depends on your installation method. For most Linux
-				based systems, the file can be found at `/etc/vector/vector.toml`.
+				based systems, the file can be found at `/etc/vector/vector.yaml`.
```

Comment:
I think we should mention the deprecated behavior too. That if the `yaml` file doesn't exist, it'll look for a `toml` one.